android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetConferenceOptions.java

```diff
@@ -98,6 +98,17 @@ public Builder() {
             featureFlags = new Bundle();
         }
 
+        /**
+         * Sets the default remote name.
+         * @param defaultRemoteDisplayName - Remote name to be displayed.
+         * @return - The {@link Builder} object itself so the method calls can be chained.
+         */
+        public Builder setDefaultRemoteDisplayName(String defaultRemoteDisplayName) {
```

Comment:
Please drop this, people can use the one liner. We don't want to add an extra function for each config value.